frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx

```diff
@@ -2,11 +2,11 @@ import * as React from 'react';
 import { FieldLevelHelp } from '@console/internal/components/utils';
 
 export const EfficiencyItemBody: React.FC<EfficiencyItemBodyProps> = React.memo(
-  ({ title, infoText, isLoading, error, getStats }) => {
+  ({ stats, title, infoText, isLoading, error, getStats }) => {
     let status: React.ReactElement;
     if (isLoading) {
       status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
-    } else if (error) {
+    } else if (error || stats === 0) {
```

Comment:
Noobaa compresion ratio can be negative certain times
```suggestion
    } else if (error || stats <= 0) {
```